packages/components/src/components/vertical-tab/vertical-tab.tsx

```diff
@@ -140,7 +142,7 @@ const VerticalTab = ({
                 </div>
             )}
 
-            {is_sidebar_enabled && (
+            {is_sidebar_enabled && isDesktop && (
```

Comment:
why are we using `isDesktop` from `@deriv-com/ui `here and `is_tablet` from `ui-store` in `packages/account/src/Containers/Account/page-overlay-wrapper.tsx `?

Reply:
Hi @amina thanks for highlighting 🙇 : 
I used `is_tablet` because we are already using `is_mobile` from ui store in below mentioned file.
`packages/account/src/Containers/Account/page-overlay-wrapper.tsx` 


and I saw in trader's hub PR they are using `useDevice` from `deriv-com/ui`  that's why I used `useDevice`  in below mentioned file 
`packages/appstore/src/modules/traders-hub/index.tsx `


which one do you recommend to use ?

Reply:
useDevice from deriv-com/ui has the new breakpoint

Reply:
Done